pkg/config/types/compute.go

```diff
@@ -62,3 +63,20 @@ type LocalPublisherConfig struct {
 	Port      int    `yaml:"Port"`
 	Directory string `yaml:"Directory"`
 }
+
+type ComputeControlPlaneConfig struct {
```

Comment:
Naming is hard. Think `ComputeHeartbeatConfig` might make more sense? or `ManagmentClientConfig` since that's the constructor that accepts this?